changes/unreleased/4875.29srP6fENz7FrGGoWSkPNa.toml

```diff
@@ -0,0 +1,5 @@
+bugfixes = "Fix no running event loop error on Python 3.14"
+[[pull_requests]]
+uid = "4875"
+author_uid = "Copilot"
```

Comment:
```suggestion
author_uid = "harshil"
```

Reply:
not currently, no. I could add that. based on commit-count and effort I'm strongly in favor of crediting you :D